warehouse/macros/define_gtfs_guidelines.sql

```diff
@@ -209,7 +210,83 @@
 {% endmacro %}
 
 {% macro organization_has_contact_info() %}
-"The organization has contact information on the website."
+"A technical contact email address is listed on the organization website"
+{% endmacro %}
+
+{% macro shapes_accurate() %}
+"Shapes in shapes.txt are precise enough to show the right-of-way that the vehicle uses and not inaccurately exit the right-of-way"
+{% endmacro %}
+
+{% macro data_license() %}
+"Includes an open license that allows commercial use"
+{% endmacro %}
+
+{% macro authentication_acceptable_schedule() %}
+"GTFS Schedule feed requires easy (if any) authentication"
+{% endmacro %}
+
+{% macro authentication_acceptable_vp() %}
+"Vehicle positions feed requires easy (if any) authentication"
+{% endmacro %}
+
+{% macro authentication_acceptable_tu() %}
+"Trip updates feed requires easy (if any) authentication"
+{% endmacro %}
+
+{% macro authentication_acceptable_sa() %}
+"Service alerts feed requires easy (if any) authentication"
+{% endmacro %}
+
+{% macro stable_url_schedule() %}
+"GTFS Schedule feed is published at a stable URI (permalink) from which it can be “fetched” automatically by trip-planning applications"
+{% endmacro %}
+
+{% macro stable_url_vp() %}
+"Vehicle positions feed is published at a stable URI (permalink) from which it can be “fetched” automatically by trip-planning applications"
+{% endmacro %}
+
+{% macro stable_url_tu() %}
+"Trip updates feed is published at a stable URI (permalink) from which it can be “fetched” automatically by trip-planning applications"
+{% endmacro %}
+
+{% macro stable_url_sa() %}
+"Service alerts feed is published at a stable URI (permalink) from which it can be “fetched” automatically by trip-planning applications"
+{% endmacro %}
+
+{% macro grading_scheme_v1() %}
+"Passes Grading Scheme v1"
```

Comment:
FWIW I feel like this is gonna be all null for a very long time 😅

Reply:
Lol yes